x/interchainquery/client/cli/tx.go

```diff
@@ -25,55 +24,15 @@ func GetTxCmd() *cobra.Command {
 	}
 
 	cmd.AddCommand(
-		QueryBalanceCmd(),
+		//QueryBalanceCmd(),
 		// TODO(TEST-53) remove cli access to ICQ queries pre-launch
 		SubmitQueryResponse(),
```

Comment:
We can remove this now

Reply:
Good call, removed. Also, closed out TEST-80 and TEST-84 (tracking updates to this code)